src/openapi.json

```diff
@@ -732,6 +732,15 @@
             "schema": {
               "type": "integer"
             }
+          },
+          {
+            "name": "includeFieldsAndValues",
+            "in": "query",
+            "description": "Include the application form fields and the associated proposal values when 'true'. This necessarily includes all proposal versions because form fields are related to a particular proposal version.",
```

Comment:
```suggestion
            "description": "Include the application form fields and the associated proposal values when `true`. This necessarily includes all proposal versions because form fields are related to a particular proposal version.",
```



This risks appearing like finding fault just to find fault, lol, but I really like using literal backticks in place of the single-quotes around `true` in the description. I doubt Swagger will actually format that as `<code></code>` — aka I don't think it's doing Markdown transformations — but it's a nice way to inline code, boolean values, etc.

Reply:
This is a great suggestion, let me try it out and see what works best with OpenAPI!

Reply:
Ooh, it sure does, neat!

<img width="394" alt="image" src="https://user-images.githubusercontent.com/4731/218816369-b0d66622-636b-4e11-bfc2-53879de50bcf.png">